sys/mips/rootdev.c

```diff
@@ -2,73 +2,168 @@
 #include <sys/klog.h>
 #include <mips/malta.h>
 #include <mips/mips.h>
+#include <mips/context.h>
 #include <mips/interrupt.h>
 #include <sys/bus.h>
 #include <sys/exception.h>
-#include <sys/pci.h>
+#include <sys/interrupt.h>
 #include <sys/devclass.h>
 
 typedef struct rootdev {
-  void *data;
+  rman_t mem, irq;
+  intr_event_t intr_event[8];
```

Comment:
We should adapt the `NIRQ` constant from `aarch64 rootdev`.

Reply:
Introduced `MIPS_NIRQ` constant.